WooCommerce/Classes/ViewRelated/Dashboard/Reviews/ReviewsDashboardCard.swift

```diff
@@ -38,20 +40,26 @@ struct ReviewsDashboardCard: View {
                 .redacted(reason: viewModel.syncingData ? [.placeholder] : [])
                 .shimmering(active: viewModel.syncingData)
             Divider()
+                .shimmering(active: viewModel.syncingData)
 
-            ForEach(Array(dummyData.enumerated()), id: \.element.reviewID) { index, review in
-                ReviewRow(for: review, isLastItem: index == dummyData.count-1)
+            if viewModel.data.isNotEmpty {
+                ForEach(Array(viewModel.data.enumerated()), id: \.element.review.reviewID) { index, reviewViewModel in
```

Comment:
The use of Array and enumerated here is so that we can get both `index` and `reviewViewModel` in the closure.

ref: https://stackoverflow.com/a/62589546